models/src/main/thrift/content/v1.thrift

```diff
@@ -11,6 +11,14 @@ struct CapiDateTime {
      * Date times are represented as i64 - epoch millis
      */
     1: required i64 dateTime
+
+    /*
+     * Also represent as a string (yyyy-MM-dd`T`HH:mm:ss.SSSZZ) in order to preserve timezone info.
+     *
+     * Note: this field makes the i64 representation redundant.
+     */
+    2: required string iso8601
```

Comment:
Note that I'm adding a new required field here. We'll need to upgrade `content-api-models` in CAPI and start sending the field in responses before we upgrade the models in the Scala client.